theme/_sass/volta/_layout.scss

```diff
@@ -151,35 +148,36 @@ main.home {
     }
 
     h1 {
-      font-family: Satisfy, cursive;
-      font-size: 3rem;
-      color: white;
-      padding: 0.5em 0;
-      border-radius: 4px;
+      padding: 0.5em 0 0 0;
       background-position: top center;
-      background-image: url('/assets/lightning.jpg');
+      background-image: url('/assets/wordmark.jpg');
       background-repeat: no-repeat;
-      background-size: 10em;
-      width: 10em;
+      background-size: 338px 150px;
+      width: 338px;
+      height: 150px;
+    }
+
+    h1, .description {
+      margin: 0 auto;
     }
 
-    h1, .description, .action {
-      margin: 1.8rem auto;
+    .action {
+      margin: 2rem auto;
     }
 
     .description {
       max-width: 35rem;
-      font-family: 'Contrail One';
+      font-family: $homepage-font;
       font-size: 1.6rem;
       line-height: 1.3;
-      color: $color-primary;
+      color: $color-primary-dark;
     }
 
     .action-button {
       display: inline-block;
       font-size: 1.2rem;
       color: #fff;
-      background-color: $color-primary;
+      background-color: $color-primary-light;
```

Comment:
This one needs to be a bit darker. If converted to HSL, 20% less "lightness" gets it there easily while keeping the look. Using `$color-primary` also works but ends up being fairly dark.

Happily, this is the only *contrast* issue!

Reply:
Thanks for all the tips on Discord today. I think we should be in good shape for color contrast now!